ssv/proposer.go

```diff
@@ -190,29 +193,21 @@ func (r *ProposerRunner) ProcessPostConsensus(signedMsg *types.SignedPartialSign
 		copy(specSig[:], sig)
 
 		if r.decidedBlindedBlock() {
-			data, err := r.GetState().DecidedValue.GetBellatrixBlindedBlockData()
```

Comment:
not sure how significant it is but versioned block is "wasteful" since ssz will fill in zero data in some fields

Reply:
If I understand your comment correctly, it doesn't make sense because we are only marshaling SSZ for specific blocks like Bellatrix, Capella, etc., rather than the entire VersionedBlock